src/main/scala/com/cloudant/clouseau/IndexService.scala

```diff
@@ -117,6 +121,11 @@ class IndexService(ctx: ServiceContext[IndexServiceArgs]) extends Service(ctx) w
         pendingSeq = newSeq
         logger.debug("Pending sequence is now %d".format(newSeq))
         'ok
+      case SetPurgeSeqMsg(newSeq: Long) =>
+        pendingPurgeSeq = newSeq
+        logger.debug("Pending purge sequence is now %d".format(newSeq))
+        commit(pendingSeq, pendingPurgeSeq)
```

Comment:
OK. I think I can modify it later.
Let me make sure that we can remove the value `pendingPurgeSeq`, right? If the `dreyfus` calls `get_purge_seq`, `clouseau` will return `purgeSeq` directly(now this value maybe is not committed yet).